assets/scss/header.scss

```diff
@@ -101,8 +105,27 @@
   padding-bottom: 20px;
   min-height: 80px;
 }
+
 .nhsuk-header--transactional {
   .nhsuk-header__search {
     height: 1rem;
   }
+  .nhsuk-header__link{
+    width: auto;
+    display: inline-block;
+    text-decoration: none;
+    height: auto;
+  }
+}
+
+.nhsuk-header__link:hover .nhsuk-logo{
+  box-shadow: none;
+}
+
+.nhsuk-header--organisation .nhsuk-header__logo{
+  .nhsuk-header__link:hover{
+
+    box-shadow: 0 0 0 4px #ffcd60;
```

Comment:
can this use the mixin color please? Ideally try to avoid putting our own hex codes in here so if the upstream changes for any reason, we automatically inherit

Reply:
Should be fixed :-)